test/proxy.test.ts

```diff
@@ -240,6 +240,43 @@ describe("", () => {
 
       expect(resText).toEqual(message);
     });
+
+    it("can proxy DELETE request", async () => {
```

Comment:
Tests seem to be pass even by commenting out the changes in utils/proxy we might add something that fails without it.